Library/Homebrew/utils/formatter.rb

```diff
@@ -0,0 +1,47 @@
+require "utils/tty"
+
+module Formatter
+  module_function
+
+  def arrow(string, color: nil)
+    if color
+      "#{Tty.send(color)}==>#{Tty.reset} #{string}"
+    else
+      "==> #{string}"
+    end
+  end
+
+  def headline(string, color: nil)
+    arrow("#{Tty.bold}#{string}#{Tty.reset}", color: color)
+  end
+
+  def success(string)
+    colorize_text_before_colon(string, :green)
+  end
+
+  def warning(string)
+    colorize_text_before_colon(string, :yellow)
+  end
+
+  def error(string)
+    colorize_text_before_colon(string, :red)
+  end
+
+  def url(string)
+    descenders_no_underline("#{Tty.blue.underline}#{string}#{Tty.reset}")
+  end
+
+  def colorize_text_before_colon(string, color)
```

Comment:
Yep, `colorize_label` or possibly a more opaque `style_label` would have worked. (I guess that method name is no longer needed given the changes you pushed since then.)